tfx/experimental/templates/taxi/e2e_tests/test_utils.py

```diff
@@ -19,18 +19,27 @@
 from __future__ import print_function
 
 import codecs
+import datetime
 import locale
 import os
+import subprocess
+import tarfile
+import time
+import urllib.request
 
 from typing import Text, List, Iterable, Tuple
 
 from absl import logging
 from click import testing as click_testing
 import tensorflow as tf
 
+from tfx.orchestration import test_utils as orchestration_test_utils
+from tfx.orchestration.kubeflow import test_utils as kubeflow_test_utils
```

Comment:
It looks like this import enforces env variables like `KFP_E2E_BASE_CONTAINER_IMAGE` and it breaks `tfx/experimental/templates/taxi/e2e_tests/beam_e2e_test.py` which should be runnable without KFP related environment variables.

How about creating a new file which contains KFP related stuff only?